updated the ui and added pagination and also added search bar and als…

DevanshBajpai09 opened the pull request:
![Screenshot 2025-06-19 003107](https://github.com/user-attachments/assets/be24b184-3fc5-4bc0-9d5e-ebd98c71bfb7)

pratik-mahalle commented:
Hey @DevanshBajpai09, Thank you for raising the pr. Our team will soon review the changes before that please make sure to add the detailed description about the pr and sign the DCO. Please make these changes

DevanshBajpai09 commented:
In this i have added more readibility and clean code with comments and also changes the ui and also added the search functionalities and also pagination with give use clear view and also well as the button design as well .
